### PR TITLE
pythonPackages.environmental-override: init at 0.1.2

### PR DESCRIPTION
**Pull request by nathanruiz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131300` at fd86958b run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.environmental-override</li>
    <li>python39Packages.environmental-override</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/environmental-override/default.nix:15:3:
```
   |
15 |   meta = {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/environmental-override/default.nix:16:0:
```
   |
16 |     description = "Easily configure apps using simple environmental overrides";
   | ^
```
</li>
  </ul>
</details>

**Review by fabaff — COMMENTED** [review]
Is this a requirement for another package? The last release was 6 years ago and there was zero action in the meantime. It looks like that open issues don't get any attention and that the module is abandoned.

**Comment by nathanruiz** [comment]
Yeah, I am attempting to package jirafs (https://github.com/coddingtonbear/jirafs), which was created by the same author.

File: pkgs/development/python-modules/environmental-override/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vhd37i6f8xh6kd61yxc2ynzgcln7v2p7fyzjmhbkdnws6gwfs6s";
+  };
+
+  meta = {
+    description = "Easily configure apps using simple environmental overrides";
+    homepage = "https://github.com/coddingtonbear/environmental-override";
+    license = lib.licenses.mit;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/environmental-override/default.nix:18]: Missing maintainer.

File: pkgs/development/python-modules/environmental-override/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+buildPythonPackage rec {
+  pname = "environmental-override";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vhd37i6f8xh6kd61yxc2ynzgcln7v2p7fyzjmhbkdnws6gwfs6s";
+  };
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/environmental-override/default.nix:14]: Are there tests we can run? If not please disable running checks.

Please add a pythonImportsCheck.

File: pkgs/development/python-modules/environmental-override/default.nix (hunk 1 above)
Comment (nathanruiz) [reply pkgs/development/python-modules/environmental-override/default.nix:23]: I've added myself as the maintainer

File: pkgs/development/python-modules/environmental-override/default.nix (hunk 2 above)
Comment (nathanruiz) [reply pkgs/development/python-modules/environmental-override/default.nix:14]: There aren't any tests, so I've set `doCheck` to `false`.

I've added a pythonImportsCheck.

**Pull request merged**